org.freedesktop.fwupd.json

```diff
@@ -0,0 +1,221 @@
+{
+  "app-id": "org.freedesktop.fwupd",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.28",
```

Comment:
Can it be moved to 3.30?

Reply:
I've pushed this change, thanks for the suggestion.